package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@autorest/typescript",
-    "version": "1.0.0-beta.3",
+    "version": "6.0.0-beta.4",
```

Comment:
Based on the previous teams and email discussion, I have changed the version to 6.0.0-beta.4 version (instead of 1.0.0-beta.4 version)